src/doc/en/reference/references/index.rst

```diff
@@ -970,6 +973,10 @@ REFERENCES:
             Anal. Appl. 15 (1994) 804-823.
             :doi:`10.1137/S0895479892230031`
 
+.. [BL1995] W. Bosma, H.W. Lenstra: Complete Systems of Addition Laws for
```

Comment:
```suggestion
.. [BL1995] W. Bosma, H.W. Lenstra: Complete Systems of Two Addition Laws for
```
I think this is the name of the paper? Unless I'm looking at the wrong thing.

Reply:
Indeed, good catch, thanks.